src/lib.rs

```diff
@@ -57,6 +57,9 @@ pub use crate::{
     value_types::ValueType,
 };
 
+#[cfg(feature = "recurrence")]
+pub use rrule::{RRuleSet, Frequency, NWeekday, RRule, Tz, Weekday};
```

Comment:
actually, maybe it is sufficient to reexport `rrule` itself. no need to make the api surface so large. accessing stuff like this is still convenient: `use icalendar::rrule::Weekday`.
That is how e.g. [axum](https://docs.rs/axum/latest/axum/) reexports [http](https://docs.rs/axum/latest/axum/#reexports) .